cdap-ui/app/features/tracker/controllers/usage-ctrl.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+class TrackerUsageController {
+  constructor($state, $scope, myTrackerApi) {
+
+    this.$state = $state;
+    this.$scope = $scope;
+    this.myTrackerApi = myTrackerApi;
+
+    let entitySplit = this.$state.params.entityType.split(':');
```

Comment:
either way the page would be broken I think, that mean the user will be navigating to an invalid state